### ISSUE TITLE
UI Color

### ISSUE THREAD

**Issue by Vldm11r** [perfect-things/perfect-home#72]
I suggest making a root color palette in code:
```
:root {
  --color1: #123;
  --color2: #000000;
  --color3: #ff7f007f;
  --color4: rgba(255,123,0,0.5);
  --bookmarkfilt:hue-rotate(0deg)brightness(1.0)saturate(1.0) !important;  
  --bookmarkfiltplus:hue-rotate(0deg)brightness(1.0)saturate(1.0) !important;
  --ico: url('https://img.png');
}
```
Color can only be replaced here, and in the rest of the code, the entire background will change for all elements, such as background.

![2020-03-10_200107](https://user-images.githubusercontent.com/59932374/76344901-02252400-630b-11ea-8069-b1d4c63e23cf.png)

**Comment by Vldm11r** [comment]
I think such a method would simplify the creation of themes in the future.

**Comment by tborychowski** [comment]
The palette is already there, albeit not in form of css variables, but stylus (css preprocessor) variables.
So it should not be too difficult to transpose them.

**Comment by tborychowski** [comment]
Available in v0.9.0

**Comment by Vldm11r** [comment]
:root {
--color-border: 
#fff;
	--color-background: 
#E3E4E6;
	--color-background-alt: 
#E3E4E6ea;
	--color-background-light: 
#F5F6F7ea;
	--color-text: 
#333444ff;
	--color-text-dimmed: 
#FFF;
	--color-highlight: 
#0099ff;
	--color-accent: 
#0099ff;
	--color-info: 
#ffffff99;
	--color-success: 
#00990099;
	--color-warning: 
#ff0000;
	--color-danger: 
#99000099;
	-----color-shadow: 
#ffffff;
}

.bookmarks .item {
	text-align: left;
	color: #333444ff;
	border-radius: 10px;
	width: var(--icon-width);
	height: var(--icon-height);
	filter: drop-shadow(  1px 1px 1px #ffffff) ;
}

.settings-pane {
	box-shadow: 0px 0 30px var(-----color-shadow);
}

select {
	background-color: #F5F6F7;
}

.select-wrap, textarea, input[type="text"], input[type="color"], input[type="number"] {
	background-color: var(--color-background-light);
color: var(--color-text);
}

.modal-backdrop.visible {
	background-color: #E3E4E6ea;
}

.modal {
	background: var(--color-background-light);
	color: var(--color-text);
	border-radius: 3px;
	margin-top: -60px;
	opacity: 100;
	transform: scale(.8);
	transition: opacity .25s,transform .25s;
	box-shadow: 0 0px 40px 20px var(-----color-shadow);
}

.context-menu {
	background: var(--color-background-light);
	color: var(--color-text);
	box-shadow: 0 0px 20px 0px var(-----color-shadow);
}


.bookmarks .item-thumb,
a[href*='photo-'], span[style*='image: url'], a[style*='image: url'], div[style*='image: url'] 
{
filter: opacity(0.5); transition:0.6s;
}

.bookmarks .item-thumb
:hover, a[href*='photo-']:hover, span[style*='image: url']:hover, a[style*='image: url']:hover, div[style*='image: url']:hover  
{
filter: opacity(1) drop-shadow( 0px 10px 15px #ffffff) ;transition:0.1s;
}
